pubspec.yaml

```diff
@@ -24,9 +25,5 @@ dependency_overrides:
       ref: issues/829-nullsafety
   flutter_map_marker_popup:
     git:
-      url: git://github.com/bsneider/flutter_map_marker_popup.git
-      ref: nullsafety
-  flutter_image:
```

Comment:
is flutter_image not needed any more, or perhaps now it is null safe?

Reply:
It is null safe now :)